client/my-sites/promote-post/components/posts-list-banner/index.tsx

```diff
@@ -23,7 +23,7 @@ export default function PostsListBanner() {
 				</div>
 				<div className="posts-list-banner__description">
 					{ translate(
-						'Reach more people promoting your work to the larger WordPress.com community of blogs and sites.'
+						'Reach more people by promoting your work to the larger WordPress.com community of blogs and sites.'
```

Comment:
No need for `hasTranslation` check as we release /advertising only to EN sites for now.